.travis.yml

```diff
@@ -94,6 +94,9 @@ env:
     - "TESTDIR=Java/wizzardo-http"
     - "TESTLANG=JavaScript"
     - "TESTLANG=Kotlin"
+    - "TESTDIR=Lisp/ningle"
+    - "TESTDIR=Lisp/ninglex"
+    - "TESTDIR=Lisp/racket"
```

Comment:
Could you change these three lines to the single line:

```
      - "TESTLANG=Lisp"
```

to utilize the docker cache and speed up testing. I'll try and do a thorough review of this PR this week. Thanks!